lib/search.js

```diff
@@ -118,9 +123,16 @@ async function processFirstPage (html, opts, savedApps, mappings) {
   return checkFinished(opts, [...savedApps, ...apps], token);
 }
 
+function isMoreSection (section) {
+  const sectionTitle = R.path(SECTIONS_MAPPING.title, section);
+  return R.is(String, sectionTitle) && !R.isEmpty(sectionTitle);
+}
+
 const INITIAL_MAPPINGS = {
   app: ['ds:4', 0, 1, 0, 23],
-  sections: ['ds:4', 0, 1]
+  sections: ['ds:4', 0, 1],
+  aboutResultSection: ['ds:4', 0, 1],
+  aboutResultsTitle: ['ds:4', 0, 1, 0, 31, 0]
```

Comment:
It looks like `aboutResultsTitle` has the same mapping as `sections`. Why do you have to duplicate the element?

Reply:
Also, the `aboutResultsTitle` should be part of a SECTION mapping, not the initial mappings. It's something specific to one section.

Reply:
as suggested its now using the section mapping + the result title mapping was moved to section specific mapping